setup.py

```diff
@@ -93,6 +93,7 @@ def find_version(*file_paths):
 
 
 setup(
+    python_requires='>=3.7',
```

Comment:
If we define it here, I think we have to remove from `setup.cfg`: https://github.com/cylc/cylc-flow/blob/d6cb92c90426ac0a310ef3ebde682c8535993165/setup.cfg#L48

Reply:
Ahh, we are setting it, good.